src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -151,15 +151,15 @@ const SendFunds = ({ initialValues, onClose, onNext, recipientAddress, selectedT
                             <Paragraph
                               className={classes.selectAddress}
                               noMargin
-                              onClick={() => setSelectedEntry(null)}
+                              onClick={() => setSelectedEntry({ address: '', name: 'string' })}
```

Comment:
Something in this component is not handled correctly.

If I focus on address input and I don't set any address and then I fulfill all the other inputs, Review button is set active. If I try to go to next step the app breaks because selectedEntry is null

Maybe is a problem with the input component.

I suspect this also happen in SendCollectible component but I can't really test it because I don't have any collectibles.

But at least the Review button is activated even no address is selected

Reply:
![image](https://user-images.githubusercontent.com/16622558/92121532-6da80400-ee0b-11ea-8f3e-08926b5dd1a2.png)

hm?

Reply:
https://github.com/gnosis/safe-react/blob/c75cac0581b7d18369db3b1ce751fb17233eb82a/src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx#L112